core-backup/src/main/scala/io/aiven/guardian/kafka/backup/BackupClientInterface.scala

```diff
@@ -211,35 +357,13 @@ object BackupClientInterface {
     * @return
     *   The position of the Stream
     */
-  def splitAtBoundaryCondition(dividedPeriodsBefore: Long, dividedPeriodsAfter: Long): BackupStreamPosition =
+  def splitAtBoundaryCondition(dividedPeriodsBefore: Long, dividedPeriodsAfter: Long): Boolean =
```

Comment:
Scaladoc is wrong and probably the name is not that descriptive anymore

Reply:
Fixed